src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -37,6 +38,24 @@ def list(self):
         fsmap = mgr.get("fs_map")
         return fsmap['filesystems']
 
+    def create(self, name: str, service_spec: Dict[str, Any]):
```

Comment:
Could you please add API tests to the PR?

Reply:
Would be good to have some API tests and a frontend unit test for cephfs create service as well.

Reply:
As we agreed on the last meeting let's skip these for now for deadline sake.